fta/samples/models.py

```diff
@@ -97,14 +97,23 @@ class LabeledSample(models.Model):
 
     objects = LabeledSampleManager()
 
-    original_sample = models.OneToOneField(
-        Sample,
+    original_sample = models.ForeignKey(
+        to=Sample,
         related_name="labeled_sample",
         on_delete=models.CASCADE,
         blank=False,
         null=False,
         help_text="Link to the original sample",
     )
+
+    superseded_by = models.ForeignKey(
+        to="self",
+        related_name="labeled_sample",
+        on_delete=models.CASCADE,
```

Comment:
I don't think cascading here is needed. Why would we want to delete all previous `LabeledSample`s if we delete the current non-superseded `LabeledSample`? I think we can just do `models.DO_NOTHING` here, which would make it so that the corresponding `Sample` has no non-superseded `LabeledSample`s (aka it's not labeled).

Reply:
I was initially thinking that if the current non-superseded LabeledSample was deleted we would want to make the previous LabeledSample the new current non-superseded LabeledSample.  This is simpler option.  To the end user it would appear that the Sample is not labeled and the history prior to the import would still be there.

Reply:
Updated to use models.SET_NULL which removed the FK. 
This will result in "rolling back" the LabeldSample import to the previous value.